ui/src/experiments/components/ExperimentAssociatedArticles.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Link } from 'react-router-dom';
+import { LITERATURE } from '../../common/routes';
+import { getPapersQueryString } from '../utils';
+
+function ExperimentAssociatedArticles({ recordId, legacyName }) {
```

Comment:
`ExperimentAssociatedArticlesLink` might be better when I saw it in another component to be used I thought it was a list component displaying the articles.